### PR TITLE
New draft versions

### PR DESCRIPTION
**Pull request by VMRuiz**
- Auto formatted code with Black for consistency
- Added support for Draft3, Draft6 and Draft7. Set Draft4 as default is not schema is provided for backward compatibility.
-Added new tests and improved coverage.

### CODE REVIEW

File: scrapy_jsonschema/pipeline.py [hunk 1]
```diff
@@ -30,14 +30,13 @@ def process_item(self, item, spider):
...
             paths_messages.append((path, error.message))
         if errors:
-            error_msg = ''
+            error_msg = ""
             for path, message in paths_messages:
-                error_msg += u'{}: {}\n'.format(path, message)
-            raise DropItem(u'schema validation failed: \n {}'
-                           .format(error_msg))
+                error_msg += u"{}: {}\n".format(path, message)
+            raise DropItem(u"schema validation failed:\n{}".format(error_msg))
```
Comment (alexander-matsievsky) [inline scrapy_jsonschema/pipeline.py]: @VMRuiz We use these error messages to analyze items' drop reasons, e.g.:

```
i = i.join(pd.io.json.json_normalize(i._DEBUG.str['drop_reason'].apply(lambda xs: {f'_DROP_REASON_{i}': x.strip() for i, x in enumerate(xs) if x and x.strip()} if isinstance(xs, list) else {})))
```

Should we switch from `schema validation failed:\n{}: {}\n{}: {}\n` format to `schema validation failed: <indented_json>`? It'll be both human- and machine-readable that way.

**Comment by alexander-matsievsky** [comment]
@VMRuiz Great work reviving this project, Victor!

File: scrapy_jsonschema/pipeline.py (hunk 1 above)
Comment (VMRuiz) [reply scrapy_jsonschema/pipeline.py]: That would be a good idea but we can't introduce non-backward compatible changes without a warning first

File: scrapy_jsonschema/__init__.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
 from scrapy_jsonschema.item import JsonSchemaItem
 from scrapy_jsonschema.pipeline import JsonSchemaValidatePipeline
 
-__version__ = '0.1.0'
+__version__ = "0.2.0"
```
Comment (kmike) [inline scrapy_jsonschema/__init__.py]: usually it is better to keep version bumps separate from feature pull requests

**Pull request closed without merging**